VMAccess/vmaccess.py

```diff
@@ -56,49 +57,70 @@ def main():
         elif re.match("^([-/]*)(update)", a):
             update()
 
+
 def install():
     hutil = Util.HandlerUtility(waagent.Log, waagent.Error, ExtensionShortName)
     hutil.do_parse_context('Uninstall')
-    hutil.do_exit(0,'Install','Installed','0', 'Install Succeeded')
+    hutil.do_exit(0, 'Install', 'Installed', '0', 'Install Succeeded')
+
 
 def enable():
     hutil = Util.HandlerUtility(waagent.Log, waagent.Error, ExtensionShortName)
     hutil.do_parse_context('Enable')
     try:
-        protect_settings = hutil._context._config['runtimeSettings'][0]['handlerSettings'].get('protectedSettings')
+        public_settings = hutil._context._config['runtimeSettings'][
+            0]['handlerSettings'].get('publicSettings')
+        protect_settings = hutil._context._config['runtimeSettings'][
+            0]['handlerSettings'].get('protectedSettings')
         reset_ssh = protect_settings.get('reset_ssh')
-        # check port each time the VM boots up
+        check_disk = public_settings.get('check_disk')
+        repair_disk = public_settings.get('repair_disk')
+    # check port each time the VM boots up
         if reset_ssh:
             _open_ssh_port()
             hutil.log("Succeeded in check and open ssh port.")
-        hutil.exit_if_enabled()
-        remove_user = protect_settings.get('remove_user')
+            hutil.exit_if_enabled()
+            remove_user = protect_settings.get('remove_user')
```

Comment:
this fix breaks... :-)
`remove_user` is used below in the if statement. I think these two lines were intended to be at the original indent level.